pkgs/development/python-modules/cfv/default.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
+  pname = "cfv";
+  version = "3.1.0";
+
```

```suggestion
  pyproject = true;
```
Even packages without a pyproject.toml should use the new builder.